tests/log-in.test.js

```diff
@@ -0,0 +1,43 @@
+import LoginPage from '../pageobjects/login.page'
+
+describe('Banking', () => {
+
+    it(' Should access to homepage with correct username and password ', async () => {
+        await LoginPage.openWeb();
+
+        await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+
+        await expect(await browser.getUrl()).to.equal('http://localhost:8080/bank/home');
+    });
+    
+    it(' Should redirect to homepage when accessing login page with a "remember me" option ', async () => {
+        await LoginPage.openWeb();
+
+        await (await $('#remember-me')).click();
+
+        await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+
+        await browser.newWindow('http://localhost:8080/bank/');
```

Comment:
see if you can use browser's baseUrl attribute for building this string, instead of hardcoding the first part of this URL (therefore decoupling your test from the specific environment the application is running on)